### PR TITLE
navigation created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
the content earlier in this branch was wrong therefore I created the whole component again.

https://user-images.githubusercontent.com/73058928/153141654-921ff950-61ce-43c4-8a90-779f03f6393f.mp4

### CODE REVIEW

File: COMPONENTS/navigation/index.css [hunk 1]
```diff
@@ -1,23 +1,123 @@
...
-  color: white;
+  top: var(--header-height);
+  right: -100%;
+  justify-content: space-between;
+  align-items: flex-start;
+  transition: ease 500ms;
+  flex-direction: column;
+  background-color: rgb(184, 184, 184);
+  padding: 1.2rem 1.8rem;
+  gap: 16px;
```
Comment (dikshadhote) [inline COMPONENTS/navigation/index.css:32]: if you are using rem, have consistency don't change units in between. Use px for values less than 10px

File: COMPONENTS/navigation/index.css [hunk 2]
```diff
@@ -1,23 +1,123 @@
...
+  color: black;
+}
+
+.header_links--active {
+  right: 25px;
```
Comment (dikshadhote) [inline COMPONENTS/navigation/index.css:37]: same as above

File: COMPONENTS/navigation/index.css [hunk 3]
```diff
@@ -1,23 +1,123 @@
...
+.header_toggle {
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
   align-items: center;
-  transition: ease 400ms;
-  cursor: default;
+  background-color: transparent;
+  width: 22.5px;
+  height: 18px;
```
Comment (dikshadhote) [inline COMPONENTS/navigation/index.css:60]: same as above

File: COMPONENTS/navigation/index.css [hunk 4]
```diff
@@ -1,23 +1,123 @@
...
+
+.header_toggle--active .header_burger:nth-child(odd) {
+  position: absolute;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+.header_toggle--active .header_burger:nth-child(1) {
+  top: 50%;
+  bottom: 50%;
```
Comment (dikshadhote) [inline COMPONENTS/navigation/index.css:85]: use one unit

File: COMPONENTS/navigation/index.css [hunk 5]
```diff
@@ -1,23 +1,123 @@
...
+  transform: rotate(45deg);
+}
+.header_toggle--active .header_burger:nth-child(3) {
+  top: 50%;
+  bottom: 50%;
```
Comment (dikshadhote) [inline COMPONENTS/navigation/index.css:90]: same as above

**Review by dikshadhote — CHANGES_REQUESTED** [review]
Use proper units and you are good to go !

**Comment by PrashanthreddyAinala** [comment]
In some places you are using rem and in some places px, be consistence other than that everything looks good for me.

File: COMPONENTS/navigation/index.css (hunk 2 above)
Comment (ivinayakg) [reply COMPONENTS/navigation/index.css:37]: this is custom i didn't want the value to change according to root font size but will try this as for your suggestion

**Pull request merged**